src/AppBundle/ValueObject/SHA1.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace AppBundle\ValueObject;
```

Comment:
We have different types of VO classes in this namespaces... Otherwise we have to move the VO classes in separate explicit namespaces...